tests/pipeline/resources/pipeline-with-short-topics/defaults.yaml

```diff
@@ -0,0 +1,34 @@
+kubernetes-app:
+  name: "${component_type}"
+  namespace: example-namespace
+
+kafka-app:
+  app:
+    streams:
+      brokers: "${broker}"
+      schema_registry_url: "${schema_registry_url}"
+  version: "2.4.2"
+
+streams-app: # inherits from kafka-app
+  app:
+    streams:
+      brokers: test
+  to:
+    topics:
+      output-topic:
+        type: 
+      error-topic:
+        type: error
+      extra-topic:
+        role: role
+  from:
+    topics:
+      input-topic:
+        role: ~
```

Comment:
what kind of role is that?

Reply:
Equal to `null`

Reply:
Hm, then I would just put `type: input` there because this is quite confusing

Reply:
I did it to ensure that if such definitions ever stop working we would know. Should I comment an explanation or replace with `type: input`

Reply:
Because it is a side-effect, I wouldn't even have it tested

Reply:
Done